File: algorithms/1_easy/N/01450_number_of_students_doing_homework_at_a_given_time/solution.cpp
class Solution {
public:
    int busyStudent(vector<int>& startTime, vector<int>& endTime, int queryTime) {
        const size_t len{startTime.size()};
        int count{0};

        for (size_t i{0}; i < len; ++i) {
            if (startTime[i] <= queryTime && queryTime <= endTime[i]) {
                count += 1;
            }
        }

        return count;
    }
};

File: algorithms/1_easy/N/01450_number_of_students_doing_homework_at_a_given_time/solution.rs
impl Solution {
    pub fn busy_student(start_time: Vec<i32>, end_time: Vec<i32>, query_time: i32) -> i32 {
        let len = start_time.len();
        let mut count = 0_i32;

        for i in 0..len {
            if start_time[i] <= query_time && query_time <= end_time[i] {
                count += 1;
            }
        }

        count
    }
}
